src/main/java/com/thingy/model/Todo.java

```diff
@@ -0,0 +1,23 @@
+package com.thingy.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Todo {
+  @Id
+  private ObjectId _id;
```

Comment:
Why _id I guess we can give simply id as well

Reply:
Very nice question. I see that _id is a mandatory field for all documents in MongoDB. But I can still name it as `id` in the java class. Refer `How the '_id' field is handled in the mapping layer
` section in https://docs.spring.io/spring-data/mongodb/docs/1.3.3.RELEASE/reference/html/mapping-chapter.html